tasks/mesh_conf.yml

```diff
@@ -51,6 +51,8 @@
     line: "{{ lookup('template', 'policy.j2') }}"
```

Comment:
This is odd.  It seems to me that the current implementation is equivalent to
```
line: "{{ item.cidr }}"
```
I'm not sure why there is a template with only `{{ item.cidr }}` in it.  Perhaps there used to be more.

I'm also not sure how this will be idempotent, and I'm not sure how it will deal with multiline ansible_managed values (but apparently `lineinfile` is working with a multiline `line` value??)

Reply:
Here's multiline comment example:
```
==> mesh.conf <==
#
# WARNING: This file is managed by ANSIBLE.
# template: libreswan-mesh.conf.j2
# modified on: 2022-02-03 15:37:46
# by user: nhosoi
# from ansiblemaster: localhost.localdomain
#
conn private
      type=tunnel
      left=%defaultroute

==> policies/clear <==
#
# WARNING: This file is managed by ANSIBLE.
# template: policy.j2
# modified on: 2022-02-03 17:40:53
# by user: nhosoi
# from ansiblemaster: localhost.localdomain
#
198.51.100.0/24

==> policies/private <==
#
# WARNING: This file is managed by ANSIBLE.
# template: policy.j2
# modified on: 2022-02-03 17:40:53
# by user: nhosoi
# from ansiblemaster: localhost.localdomain
#
203.0.113.0/24

==> policies/private-or-clear <==
#
# WARNING: This file is managed by ANSIBLE.
# template: policy.j2
# modified on: 2022-02-03 17:40:53
# by user: nhosoi
# from ansiblemaster: localhost.localdomain
#
10.0.2.0/24

169.254.0.0/16
```
`lineinfile` looks working as we wish...